### DISCUSSION TITLE
Confusion regarding "Pull Shark" achievement

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18190] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm a bit confused regarding the "Pull Shark" achievement. It claims I've only opened 2 PR's that have been merged, however, that couldn't be further from reality.

Initially I thought that this was maybe just limited to public repositories, so I checked out a co-workers page, and surprisingly the PR's that he has made to the exact same private repository that I have made a bunch of PR's to were taken into account. Is this a bug or am I misunderstanding something?

I can absolutely assure you that I've merged more than 2 PR's to the `master` / `main` branch since I've started using Github in 2016. ([Example 1](https://github.com/prisma/prisma/pull/1248), [Example 2](https://github.com/js-org/js.org/pull/5740))

### My achievement
![image](https://user-images.githubusercontent.com/21268739/172954563-9939ac37-7fd3-4590-a243-210f99724511.png)

### Coworkers achievement
_Viewed from incognito mode to hide repository name._

![image](https://user-images.githubusercontent.com/21268739/172954435-86b10358-da4d-47f3-9361-79ce7c526fde.png)

**Comment by ElijahPepe** [comment, marked as answer]
The Pull Shark achievement is ranked by how many pull requests you've merged; x1 signifies that you've merged at least 2 pull requests, x2 signifies that you've merged at least 16 pull requests, and x3 signifies that you've merged at least 128 pull requests.

**Reply by ghost** [reply]
Thanks! That makes sense

**Reply by Zeex** [reply]
Isn't it supposed to count pull requests that you opened and someone else have merged?
"Name> opened pull requests that have been merged"
Or am I reading it wrong?

**Reply by lisenet** [reply]
@Zeex you can merge your own pull requests.

**Reply by kaveesha-sithum** [reply]
@Zeex you can merge your own pull requests.

**Reply by thadguidry** [reply]
@ElijahPepe if it's an achievement to reward folks that **review and merge code** and not **open pull requests** ... then the description of Pull Shark is wrong.  It should be described as "@somebody merged pull requests" or "@somebody reviewed and merged pull requests".

**Comment by alfiedotwtf** [comment]
So does that mean x4 @ 65536? I think I speak for @thecodeah and expected that xN should be where N = the number of actual PRs merged

**Reply by lisenet** [reply]
@alfiedotwtf x4 @ 1024 I think.

x1 - 2 PRs
x2 - 16 (2 * 8) PRs 
x3 - 128 (2 * 8 * 8) PRs 
x4 - 1024 (2 * 8 * 8 * 8) PRs 
x5 - 8192 (2 * 8 * 8 * 8 * 8) PRs 
and so on...

**Reply by alfiedotwtf** [reply]
:facepalm:  that was a brain fart on my part... for some reason my brain was thinking:

x1 - bit
x2 - nibble
x3 - byte
x4 - word
etc,

but yeah, I was off by one there

**Reply by quoideneuf** [reply]
I came here wondering the same thing :)

**Reply by bernhardreiter** [reply]
It seems that x4 is 1024, see example https://github.com/ljharb found by https://github.com/Schweinepriester/github-profile-achievements/issues/2#issuecomment-1151320396 .

**Reply by 5atoshiNakamoto** [reply]
Why does it multiply by 8 each time?

**Reply by blunderdome** [reply]
I think because Github's mascot is the Octocat.

**Comment by HBL001** [comment]
How do I turn these "achievements" off?

**Reply by lisenet** [reply]
Navigate to https://github.com/settings/profile and untick the box that says "Show Achievements on my profile".

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide

**Reply by harish18092002** [reply]
How to get starstruck badge??

**Reply by Vipul1432** [reply]
When a Repository on your account hits 16 stars, even if it was transfered, you will receive the badge of starstruck .

**Reply by SANCHIT0055** [reply]
> How to get starstruck badge??

You can get **Starstruck badge**  by gaining 16  stars in a repository.

you get default by 16 stars gained in a single repository
![image](https://github.com/user-attachments/assets/325a639b-993a-4ab4-b717-7bb341dd11a7)

also available in Bronze, Silver, and Gold achievements.

Bronze when 128 Stars gained in a single repository
![image](https://github.com/user-attachments/assets/a39654d4-b0e1-4fbd-b302-a18f7e5b0cf3)

Silver when 512 Stars gained in a single repository
![image](https://github.com/user-attachments/assets/ecf822b9-b6d7-4eb2-ad28-9a5e32167012)

Gold when 4096 Stars gained in a single repository
![image](https://github.com/user-attachments/assets/b7a4abd2-9279-4600-850f-c8a526eb59f4)

**Reply by diobenu2silva** [reply]
let's exchange stars. Please help each other.

https://github.com/diobenu2silva/pumpfun_SC
https://github.com/diobenu2silva/zeroGpt4
https://github.com/diobenu2silva/solArmy_staking_SC
https://github.com/diobenu2silva/triangular_arbitrage_multi_exchange
